### DISCUSSION TITLE
Does current version support Transmission 4.0.0 beta 1?

### DISCUSSION THREAD

**Discussion by Piddox** [6c65726f79/Transmissionic#1249] [Q&A] [answered]
See title. I don't wanna update to 4.00 if Transmissionic does not support it. But I would really like to try 4.0.0 as it seems to fix a lot of issues.
Transmission 4.0.0 beta 1 can be found here: https://github.com/transmission/transmission/releases/tag/4.0.0-beta.1

**Comment by 6c65726f79** [comment, marked as answer]
I just tested it, it works perfectly. But it seems that you have to set the TRANSMISSION_WEB_HOME environment variable because there is no longer a web folder in the transmission folder.

Edit: The `web` folder is now called `public_html` in v4

**Reply by chitalov** [reply]
How is this done ?

**Reply by 6c65726f79** [reply]
It depends on your OS, but you can easily find information about it on the internet. The name of the environment variable must be TRANSMISSION_WEB_HOME, and the value is the absolute path of the folder where the web UI is located.

**Reply by Piddox** [reply]
> How is this done ?

I'm not 100%, sure but I think for RaspberryPiOS, the location is /usr/share/transmission/web/.
The environment variable needs to be added in file /etc/init.d/transmission, by adding this line there:
TRANSMISSION_WEB_HOME=/usr/share/transmission/web/

But if @6c65726f79 can confirm that would be very helpful ^^

**Reply by 6c65726f79** [reply]
> But if @6c65726f79 can confirm that would be very helpful ^^

I think the recommended way on Linux is to edit the `~/.bash_profile` file and add this line:
```export TRANSMISSION_WEB_HOME=/usr/share/transmission/web/```

But your method probably works too

**Reply by chitalov** [reply]
I am using docker ,but I'm obviously doing something wrong.


![Screenshot 2023-02-11 232727](https://user-images.githubusercontent.com/73425550/218281926-5ce73eb8-c96e-4f69-9973-14832045939f.png)
![Screenshot 2023-02-11 232818](https://user-images.githubusercontent.com/73425550/218281932-6fcc95a4-f836-493e-b57e-e6fc65e650fe.png)

![Screenshot 2023-02-11 232748](https://user-images.githubusercontent.com/73425550/218281952-d13667d4-d02a-44ea-a826-1b2383775f1b.png)

**Reply by 6c65726f79** [reply]
I'm not very familiar with docker, but you could try to replace `TRANSMISSION_WEB_HOME=/opt/transmission/web` by `TRANSMISSION_WEB_HOME=/config/web`

**Reply by chitalov** [reply]
Same result.

**Reply by 6c65726f79** [reply]
if you have not yet succeeded, you can try [this method](https://github.com/haugene/docker-transmission-openvpn/discussions/2271#discussion-4114596):

1. Add this volume: `/opt/transmission/web:/web`
2. Edit the environment variable like this: `TRANSMISSION_WEB_HOME=/web`

**Reply by chitalov** [reply]
TRANSMISSION_WEB_HOME=/config/web
Тhis is how it works, maybe i made a typo last time.
![Screenshot 2023-02-15 230000](https://user-images.githubusercontent.com/73425550/219158894-c939a424-98a3-466f-9b47-a8698cfd84bd.png)
![image](https://user-images.githubusercontent.com/73425550/219159227-92062a28-fe15-4c89-92b4-8aa11d428d55.png)
